vietnamese/vietnamese.md

```diff
@@ -32,14 +32,21 @@
 ## Machine translation
 
 ### English-Vietnamese translation
-* Dataset is from [The IWSLT 2015 Evaluation Campaign](http://workshop2015.iwslt.org/downloads/proceeding.pdf), also be obtained from [https://github.com/tensorflow/nmt](https://github.com/tensorflow/nmt): `tst2012` is used for development while `tst2013` is used for test. Scores are computed for single models.
+* Dataset is from [The IWSLT 2015 Evaluation Campaign](http://workshop2015.iwslt.org/downloads/proceeding.pdf), also be obtained from [https://github.com/tensorflow/nmt](https://github.com/tensorflow/nmt).
 
-#### English-to-Vietnamese
+##### English-to-Vietnamese & `tst2015` is used for test
```

Comment:
```suggestion
##### English-to-Vietnamese
`tst2015` is used for test
```

This ensures that the links do not break in web rendering. This helps readers navigate and discover your contributions much more easily.